docs/create-user.rst

```diff
@@ -0,0 +1,76 @@
+.. _create-user:
+
+===========
+Create user
+===========
+
+
+------------
+Introduction
+------------
+
+This part of the documentation sheds some light on the topics of `user
+management`_ and `privileges`_.
+
+CrateDB ships with a superuser account called "``crate``", which has the
+privileges to perform any action. However, with the default configuration, this
+superuser can only access CreateDB from the local machine CrateDB has been
```

Comment:
There is a small typo `CreateDB` ;)

Reply:
https://github.com/crate/crate-tutorials/pull/58 :)